src/routes/docs/contributing/code-style.svx

```diff
@@ -10,32 +10,97 @@
   
 # Code Style
 
-1. **Use** `PascalCase`:
+To make the codebase consistent and easy to understand, we require you to follow our predefined code style rules:
 
-- class names
-- method names
-- const names
+1. Naming notation for objects and functions
+- Use PascalCase for classes
+- Use PascalCase prefixed with 'I' for interfaces
+- Use PascalCase for structs
+- Use PascalCase for functions/methods
+- Use "Base" prefix for base classes
+- Use "Service" suffix for service classes and interfaces
+- Use "Model" suffix for model classes
+- Use "ViewModel" suffix for view model classes
+- Use "Dialog" suffix for dialog controls
+- Use "Control" suffix for user controls
+- Use "Page" suffix for pages
 
-2. **Use** `camelCase`:
+2. Naming notation for variables
+- Use PascalCase for private/protected/public static fields/properties
+- Use PascalCase for properties
+- Use PascalCase prefixed with '_' for private fields backing properties
```

Comment:
Assuming we are talking about the same thing, I think those fields are just called "backing fields", maybe that is more clear for folks?